Source/WebCore/page/EventHandler.cpp

```diff
@@ -424,6 +424,7 @@ void EventHandler::clear()
 #endif
     m_maxMouseMovedDuration = 0;
     m_didStartDrag = false;
+    m_touchPressed = false;
```

Comment:
This should be inside the `#if ENABLE(TOUCH_EVENTS) && !ENABLE(IOS_TOUCH_EVENTS)` a couple of lines above.

Reply:
Done. Pushed new commit with fix.